packages/client/src/arpa_reporter/views/ValidationView.vue

```diff
@@ -122,7 +122,7 @@ export default {
   computed: {
     revalidateBtnClass() {
       return {
-        'btn-primary': !this.commit,
+        'usdr-btn-primary': !this.commit,
```

Comment:
Note to reviewer: before/after screenshots

## Before

![image](https://github.com/user-attachments/assets/c32ded0f-fa76-44fc-94b3-ce004c56e40d)

## After
![image](https://github.com/user-attachments/assets/2456996e-bd12-4541-8105-4689de22c584)